R/utils.R

```diff
@@ -35,8 +35,32 @@ wrapExpr <- function(func, ...) {
 #   expandExpr(quote(!!a + !!b), list(a = quote(two)), env)
 # })
 expandExpr <- function(expr, data, env) {
-  wrappedExpr <- wrapExpr(rlang::quo, expr)
-  rlang::quo_get_expr(eval(wrappedExpr, data, env))
+
+  expr <- walk_ast(expr, function(x) {
+    expandExpr(x, data, env)
+  })
+
+  if (rlang::is_call(expr, "..", n = 1) && is.null(names(x))) {
```

Comment:
Maybe we should have a check somewhere that `..` isn't already defined by something else (and error if it is?)